BrickBrackBruck/WalterDerHalter.cs

```diff
@@ -1,23 +1,121 @@
 using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.IO;
+
 
 namespace BrickBrackBruck
 {
+
     class WalterDerHalter
     {
-        // List<
+        List<Ding> dinger = new List<Ding>();        
+
+        DateTime tag0 = new DateTime(1990,01,01);
+
+
+        public bool VerzeichnisHinzufügen(string pfad)
+        {
+            try
+            {
+                string[] Files = System.IO.Directory.GetFiles(pfad);
+
+                foreach (string f in Files)
+                {
+                    if (!DingHinzufügen( f ))
+                        return false;
+                }  
+            }
+            catch (Exception ex)
+            {
+                Console.WriteLine("Fehler (VerzeichnisHinzufügen): " + ex.Message);   
```

Comment:
das is tein guter Grund - wirf eine neue FileNotParsableException(File) weiter.
dann hat man einen Kontext, was passiert ist.
dies ist übrigens eine Regel: immer wenn interessanter Kontext hinzukommt, wirft man eine neue Exception. So kann man später gezielt Meldungen im Dialog o.ä. anzeigen